code/espurna/relay.ino

```diff
@@ -383,7 +383,7 @@ void _relayProcess(bool mode) {
     if (_relay_sync_locked && needs_unlock && changed) {
         _relaySyncUnlock();
     }
-
+	ledUpdate(true);
```

Comment:
I'd remove this then.
`LED_SUPPORT` can hard-depend on `BROKER_SUPPORT` then? see dependencies.h & led mode (which can actually be disabled, removing `ledUpdate` symbol)

Reply:
Thats correct, just making it depend on BROKER would solve the problem👍.
its ok to remove this line...